addons/pause/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Pause button",
+  "name": "Project pause button",
```

Comment:
I think short names are better for simple addons like this. You can add more detail to the description if you think it's necessary, for example "Adds a button next to the green flag that pauses the project."

Reply:
Other related addons such as "muted project player mode" and "project volume slider" have "project" in their names.

Reply:
It's obvious that "pause button" refers to projects. What else in the Scratch website could be paused?